Sources/AutomergeRepo/Repo.swift

```diff
@@ -23,6 +23,12 @@ public final class Repo {
 
     let defaultVerbosity: LogVerbosity = .errorOnly
     private var levels: [LogComponent: LogVerbosity] = [:]
+    /// LogVerbosity of Documents created from the Repo
+    public var documentLogVerbosity: LogVerbosity = .errorOnly {
+        didSet {
+            self.storage?.documentLogLevel = documentLogVerbosity
```

Comment:
`documentLogLevel` vs. `documentLogVerbosity` feels a bit odd. Happy to change either, e.g. switch the language in the storage to match what's in the repo or the other way around!